src/librustdoc/clean/mod.rs

```diff
@@ -2299,14 +2299,14 @@ impl Clean<Item> for (&hir::MacroDef<'_>, Option<Symbol>) {
             if matchers.len() <= 1 {
                 format!(
                     "{}macro {}{} {{\n    ...\n}}",
-                    vis.print_with_space(cx.tcx),
+                    vis.print_with_space(cx.tcx, item.hir_id.owner),
                     name,
                     matchers.iter().map(|span| span.to_src(cx)).collect::<String>(),
                 )
             } else {
                 format!(
                     "{}macro {} {{\n{}}}",
-                    vis.print_with_space(cx.tcx),
+                    vis.print_with_space(cx.tcx, item.hir_id.owner),
```

Comment:
Are these two correct? I'm not sure if this is the correct LocalDefId to use.

Reply:
This should be the ID of the item itself, not its owner ('owner' here is something like the containing scope). It should be `cx.tcx.hir().local_def_id(item.hir_id)` instead.